bundles/python/python_lexer.moon

```diff
@@ -46,12 +46,15 @@ howl.aux.lpeg_lexer ->
   operator = c 'operator', S'+-*/%~&^=!<>;:,.(){}[]|`'
 
   name = (alpha + '_')^1 * (alpha + digit + P'_')^0
+  user_defined_constant = c('constant', (upper + '_')^1) * -#(alpha + digit)
```

Comment:
Should this not highlight `_`, since it's used so often to discard results? e.g. `_, a = xyz`